java/BUILD.bazel

```diff
@@ -75,6 +75,7 @@ define_java_module(
         "@maven//:org_ow2_asm_asm",
         "@maven//:org_slf4j_slf4j_api",
         "@maven//:org_slf4j_slf4j_log4j12",
+        "@maven//:org_testng_testng",
```

Comment:
Is this needed? I think only the tests need testng

Reply:
In my and several other colleagues' Mac, if we don't use this, there will be compiler error when using `mvn clean install -DskipTests`. 
```
[ERROR] /Users/yuhguo/git/ray/java/runtime/src/test/java/org/ray/runtime/functionmanager/FunctionManagerTest.java:[17,18] package org.testng does not exist.
```
If we use `mvn clean install -Dmaven.test.skip` and then `mvn test`, there will the same compilation error.
FunctionManagerTest.java is the class that need compiler time testng.

Reply:
sorry, my bad. Created another PR to fix this, https://github.com/ray-project/ray/pull/4800/files